File: help/eng_gb/exercicios_da_ficha_3/Ficha03_011/Ficha03_011.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_011 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("integer number\t: " );
        int n1 = Integer.valueOf(keyb.nextLine());
        System.out.print("real number \t: " );
        double n2 = Double.valueOf(keyb.nextLine());
        System.out.print("Ordered Numbers" );
        System.out.print("\n" + Math.min( n1 , n2 ) );
        System.out.print("\n" + Math.max( n1 , n2 ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:57

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_011/Ficha03_011.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha03_011 {

    public:
        int main() {
            cout << "integer number\t: ";
            int n1;
            cin >> n1;
            cout << "real number \t: ";
            float n2;
            cin >> n2;
            cout << "Ordered Numbers";
            cout << "\n" <<( min( n1 , n2 )  );
            cout << "\n" <<( max( n1 , n2 )  );
            return 0;
        }
};// end of class Ficha03_011
// Execute main function
int main(){
    Ficha03_011 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:57
